src/aspire/basis/fb_2d.py

```diff
@@ -237,47 +237,3 @@ def evaluate_t(self, v):
         v = roll_dim(v, sz_roll)
         return v
 
```

Comment:
Hey looks like `codefactor.io` caught something here. Very nice! Let's just take what it says at face value unless we have a good reason to disagree.

Reply:
Yes. It found a tailing newline and two old issues were removed. I fixed it, rebase and pushed back.